include/EVT/io/GPIO.hpp

```diff
@@ -63,7 +72,7 @@ class GPIO {
      * @param[in] pin The pin for the GPIO instance to use.
      * @param[in] direction The directional flow of data.
      */
```

Comment:
The documentation for several of these GPIO functions needs a new @param to reflect the new parameter.